features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Base/RouteMenuMapping.jsx

```diff
@@ -214,7 +215,7 @@ const RouteMenuMapping = (intl) => [
                     defaultMessage: 'Subscription Policies',
                 }),
                 path: '/throttling/subscription',
-                component: () => <DemoTable />,
+                component: () => <SubscriptionThrottlingPolicies />,
```

Comment:
we don't  need to use lambda here right? component accept just component class/function.

component: SubscriptionThrottlingPolicies

Reply:
@kavishkafernando We have to give a lambda function here. Otherwise an error will be thrown as follows.

```
Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object. Check the render method of `Context.Consumer`.
```